src/main/java/edu/gatech/cs2340/trydent/sample/tictactoe/TitleScreenController.java

```diff
@@ -0,0 +1,155 @@
+package edu.gatech.cs2340.trydent.sample.tictactoe;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.MouseButton;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+import edu.gatech.cs2340.trydent.ContinuousEvent;
+import edu.gatech.cs2340.trydent.GameObject;
+import edu.gatech.cs2340.trydent.Keyboard;
+import edu.gatech.cs2340.trydent.Mouse;
+import edu.gatech.cs2340.trydent.OneTimeEvent;
+import edu.gatech.cs2340.trydent.TrydentEngine;
+import edu.gatech.cs2340.trydent.log.Log;
+import edu.gatech.cs2340.trydent.math.Position;
+
+
+public class TitleScreenController {
+    @FXML
+    private Button start1;
+    @FXML
+    private Button start2;
+    @FXML
+    private Button start3;
+
+    private boolean started2;
+    private boolean started3;
+    private Map<String, Rectangle> tiles;
+    private Set<Rectangle> used;
+    private boolean player;
+
+    public TitleScreenController() {
+        Log.debug("Initialized tic-tac-toe title screen.");
+        new ContinuousEvent(){
+            @Override
+            public void onUpdate() {
+                if(started2) {
+                    handleKeyboard();
+                } else if(started3) {
+                    handleMouse();
+                }
+            }
+        };
+    }
+
+    private void handleKeyboard() {
+        Rectangle target;
+        if(Keyboard.isKeyDownOnce(KeyCode.Q)){
+            target = tiles.get("0_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.A)){
+            target = tiles.get("0_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.Z)){
+            target = tiles.get("0_2");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.W)){
+            target = tiles.get("1_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.S)){
+            target = tiles.get("1_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.X)){
+            target = tiles.get("1_2");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.E)){
+            target = tiles.get("2_0");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.D)){
+            target = tiles.get("2_1");
+        } else if(Keyboard.isKeyDownOnce(KeyCode.C)){
+            target = tiles.get("2_2");
+        } else {
+            target = null;
+        }
```

Comment:
It would be cleaner if you put this logic into a HashMap lookup rather than cascading if-else statements. It looks like all you need is a Map<KeyCode, String> to cover this.